src/olympia/activity/admin.py

```diff
@@ -31,4 +32,52 @@ def has_delete_permission(self, request, obj=None):
         return False
 
 
+class ReviewActionReasonLogAdmin(admin.ModelAdmin):
+    date_hierarchy = 'created'
+    fields = (
+        'created',
+        'activity_log',
+        'user_email',
```

Comment:
you can specify the field directly here like you have in `search_fields` without adding a ` def user_email` method.

Reply:
I tried that first, but I get an error: `Unknown field(s) (activity_log__user__email) specified for ReviewActionReasonLog. Check fields/fieldsets/exclude attributes of class ReviewActionReasonLogAdmin.` I don't really understand why it works for `search_fields` but not for `fields` or `list_display`.

Reply:
mmm, I forget what's allowed.  Okay, well elsewhere we've used the full name and then defined a method with that name.  e.g. `activity_log__user__email` in `fields` and `def activity_log__user__email` in the model class.

Reply:
I was able to use the same name in `fields` and `search_fields`, by renaming my method in `admin.py` to `activity_log__user__email`, but I cannot seem to get it to work by adding a method to the model, which seems to be what you are suggesting above. I can't seem to find an example like this in any of our model files, but I do see a few examples of it being done the way I did it here, in the `admin.py` file. When I push a new commit you can take a look and let me know what you think.

Reply:
whoops, I meant `ModelAdmin` rather than `Modal`